### ISSUE TITLE
Still have METAFLOW_SERVICE_URL missing after configuration

### ISSUE THREAD

**Issue by Jeffwan** [Netflix/metaflow#43]
I follow the guidance to setup metaflow on aws, but `METAFLOW_SERVICE_URL` is not part of the configuration flow, when I check `./metaflowconfig/config.json`, it only has `METADATA_SERVICE_URL`. Seems the step in the flow set variable to `METADATA_SERVICE_URL` but not `METAFLOW_SERVICE_URL`

```
ave you setup your AWS credentials? [y/N]: y

Do you want to use AWS S3 as your datastore? [Y/n]: Y
	AWS S3
	Please enter the bucket prefix to use for your flows: metaflow3-metaflows3bucket-pxxxe
	Please enter the bucket prefix to use for your data [metaflow3-metaflows3bucket-pxxxe/data]:

Do you want to use AWS Batch for compute? [Y/n]: y

	AWS Batch
	Please enter the job queue to use for batch: arn:aws:batch:us-west-2:<account_id>:job-queue/job-queue-metaflow3
	Please enter the default container image to use:
	Please enter the default container image to use: continuumio/anaconda
	Please enter the container registry: <account_id>.dkr.ecr.us-west-2.amazonaws.com/metaflow
	Please enter the IAM role to use for the container to get AWS S3 access: arn:aws:iam::<account_id>:role/metaflow3-BatchS3TaskRole-1IXNDQD2ND1AL

Do you want to use a (remote) metadata service? [Y/n]: y
	Metadata service
	Please enter the URL for your metadata service: https://tgp9.execute-api.us-west-2.amazonaws.com/api/

Do you want to use conda for dependency management? [Y/n]: Y
	Conda on AWS S3
	Please enter the bucket prefix for storing conda packages [metaflow3-metaflows3bucket-pxxxe/conda]:

```

```
➜  metaflow-tutorials python3 00-helloworld/helloworld.py run
Metaflow 2.0.0 executing HelloFlow for user:shjiaxin
Validating your flow...
    The graph looks good!
Running pylint...
    Pylint not found, so extra checks are disabled.
    Flow failed:
    Missing Metaflow Service URL. Specify with METAFLOW_SERVICE_URL environment variable
```

**Comment by savingoyal** [comment]
Thanks for reporting the bug. @seeravikiran can you take this up?

**Comment by savingoyal** [comment]
To unblock yourself, you can set `METAFLOW_SERVICE_URL ` in `./metaflowconfig/config.json` to the actual URL.

**Comment by seeravikiran** [comment]
From offline discussion, @savingoyal plans to take this up since he's anyway revisiting the `configure aws` flow.
Thanks for reporting the bug @Jeffwan.

**Comment by rickyshelton** [comment]
PR raised for this here
https://github.com/Netflix/metaflow/pull/47

**Comment by seeravikiran** [comment]
Marking this as closed since its been fixed at HEAD.
